ui/src/authorizations/apis/index.ts

```diff
@@ -19,11 +20,15 @@ export const createAuthorization = async (
   }
 }
 
-export const getAuth0Config = async (): Promise<Auth0Config> => {
+export const getAuth0Config = async (
+  redirectTo: string
+): Promise<Auth0Config> => {
   try {
-    const response = await fetch(
-      `${getAPIBasepath()}/api/v2private/oauth/clientConfig`
-    )
+    let url = `${getAPIBasepath()}/api/v2private/oauth/clientConfig`
+    if (isFlagEnabled('redirectto')) {
+      url = `${getAPIBasepath()}/api/v2private/oauth/clientConfig?redirectTo=${redirectTo}`
```

Comment:
why not just `url += ?redirectTo=` here?

Reply:
No reason not to. Thought about it but figured this was a bit more explicit and would require less manipulation once the feature flag is removed